src/pages/recovery/recovery.js

```diff
@@ -68,7 +68,7 @@ const Recovery = () => {
             value={values.email}
             error={!!errorMsg}
           />
-          <div className={styles.errorText}>{errorMsg || ''}</div>
+          {errorMsg ? <div className={styles.errorText}>{errorMsg}</div> : null}
```

Comment:
you can use  errorMsg &&, instead of  ternary operator with null

Reply:
In React TypeScrypt its bad idea

Reply:
we don't use TypeScript